### PR TITLE
PVRE reporting for short lived instances

### PR DESCRIPTION
**Pull request by sushimaAws**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (hakuna-matatah) [inline tests/tasks/setup/eks/awscli-mng.yaml:18]: let's use EKS default version as default. 1.21 is not being supported anymore at this point.

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (hakuna-matatah) [inline tests/tasks/setup/eks/awscli-mng.yaml:91]: can we not embed this and pull it in from assets like we had discussed earlier ? Like how we do for [vpc](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/assets/amazon-eks-vpc.json)

**Comment by hakuna-matatah** [comment]
Have we verified that this template fixes our two issues that we have been having ?

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (sushimaAws) [reply tests/tasks/setup/eks/awscli-mng.yaml:18]: Did you mean to add a version here? 1.24?

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (sushimaAws) [reply tests/tasks/setup/eks/awscli-mng.yaml:91]: This LT has parameters that we are including for each run like Kubernetes version, node arn which needs to be included once the task is running. Nathan is on the same page too. We can discuss further if there are specific values that can be hardcoded? 

Or do you mean we can add this template in the assets and add these values when we curl? Let me know either ways

**Comment by sushimaAws** [comment]
I created a launch template running this CFN stack. I can validate if we can create a MNG with this LT which ideally should fix the issues. Not sure if I could validate it further

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (hakuna-matatah) [reply tests/tasks/setup/eks/awscli-mng.yaml:18]: Default version is the version EKS uses when you attempt to create a EKS cluster without mentioning a version, i think it's 1.28 now.

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (hakuna-matatah) [reply tests/tasks/setup/eks/awscli-mng.yaml:91]: >do you mean we can add this template in the assets and add these values when we curl?
This is what i meant ^^^.
Curl it like how we pull vpc and substitute values that you wanted in that file in the task if it can't be generalized. So, task is much more flexible and readable.

Comment (hakuna-matatah) [reply tests/tasks/setup/eks/awscli-mng.yaml:91]: Think of it in terms of, if I bring in another custom LT, how can I reuse this task for that so it's not tightly coupled ?

**Comment by hakuna-matatah** [comment]
> I created a launch template running this CFN stack in my dev account. If I can have permissions for perf lab, I can validate if we can create a MNG with this LT which ideally should fix the issues. Not sure if I could validate it further

I will give you access to perflab, we should verify if this task is running smoothly on experimental infra.

File: tests/assets/eks_nodeGroup_LaunchTemplate.yaml
Comment (hakuna-matatah) [inline tests/assets/eks_nodeGroup_LaunchTemplate.yaml:9]: this won't work, because we need mixed instance fleet, otherwise we would run into icing issues.

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (sushimaAws) [reply tests/tasks/setup/eks/awscli-mng.yaml]: changed the default to 1.28

File: tests/tasks/setup/eks/awscli-mng.yaml
Comment (sushimaAws) [reply tests/tasks/setup/eks/awscli-mng.yaml]: Added the LT to assets

File: tests/assets/eks_nodeGroup_LaunchTemplate.yaml
Comment (sushimaAws) [reply tests/assets/eks_nodeGroup_LaunchTemplate.yaml]: Removed instances from LT

File: tests/pipelines/eks/awscli-cl2-load-with-addons.yaml
Comment (hakuna-matatah) [inline tests/pipelines/eks/awscli-cl2-load-with-addons.yaml]: we need to remove `sushima` prefix

File: tests/tasks/setup/eks/awscli-cfn-lt.yaml
Comment (hakuna-matatah) [inline tests/tasks/setup/eks/awscli-cfn-lt.yaml:5]: could you trim `create` from the suffix ?

File: tests/tasks/setup/eks/awscli-cfn-lt.yaml
Comment (hakuna-matatah) [inline tests/tasks/setup/eks/awscli-cfn-lt.yaml:10]: let's keep the description generic please.

File: tests/pipelines/eks/awscli-cl2-load-with-addons.yaml
Comment (sushimaAws) [reply tests/pipelines/eks/awscli-cl2-load-with-addons.yaml]: I think this change is an older one. This has been removed

File: tests/tasks/setup/eks/awscli-cfn-lt.yaml
Comment (hakuna-matatah) [inline tests/tasks/setup/eks/awscli-cfn-lt.yaml:15]: `name` is extra in description

Comment (hakuna-matatah) [reply tests/tasks/setup/eks/awscli-cfn-lt.yaml:15]: And description seems bit off.

**Comment by hakuna-matatah** [comment]
Overall nice work @sushimaAws.

File: tests/tasks/setup/eks/awscli-cfn-lt.yaml
Comment (sushimaAws) [reply tests/tasks/setup/eks/awscli-cfn-lt.yaml]: Removed this

File: tests/tasks/setup/eks/awscli-cfn-lt.yaml
Comment (sushimaAws) [reply tests/tasks/setup/eks/awscli-cfn-lt.yaml]: Corrected this

**Review by hakuna-matatah — APPROVED** [review]
/lgtm
/approve

**Comment by hakuna-matatah** [comment]
Do not merge until internal changes are reviewed and merged as it will break current tests.

**Review by ashishranjan738 — APPROVED** [review]
/LGTM

**Pull request merged**
